modules/http4s-munit/src/main/scala/munit/config/Config.scala

```diff
@@ -0,0 +1,3 @@
+package munit.config
+
+final case class Config(repetitions: Option[Int], maxConcurrent: Option[Int])
```

Comment:
I think this may be too generic to live in `munit's` package, we could use `Http4sMunitConfig` to scope it just to this library

Reply:
See [`b976eac` (#42)](https://github.com/alejandrohdezma/http4s-munit/pull/42/commits/b976eacb063bb9e289c3a11cd6c8c8d2e49add84)